src/main/java/core/basesyntax/bookstore/service/impl/UserServiceImpl.java

```diff
@@ -36,6 +41,12 @@ public UserDto save(UserRegistrationRequestDto requestDto) {
                 new EntityNotFoundException("Can't find role by name. "
                         + "Name: " + Role.RoleName.USER));
         user.setRoles(Set.of(userRole));
-        return userMapper.toDto(userRepository.save(user));
+        userRepository.save(user);
+
+        ShoppingCart shoppingCart = new ShoppingCart();
```

Comment:
create method in ShoppingCartService `createShoppingCart(User user)` to create shop cart for user